### PR TITLE
Add a github actions CD tutorial

### PR DESCRIPTION
**Pull request by digitalronin**
Add a tutorial on setting up continuous deployment (CD) of a GitHub repository
to a namespace on the cloud platform, using GitHub Actions.

### CODE REVIEW

File: source/documentation/deploying-an-app/github-actions-continuous-deployment.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+repository, e.g.
+
+```
+namespaces/live-1.cloud-platform.service.justice.gov.uk/dstest
+```
+
+...and run this:
+
+```bash
+cloud-platform environment serviceaccount create
```
Comment (jasonBirchall) [inline source/documentation/deploying-an-app/github-actions-continuous-deployment.html.md.erb:311]: As this isn't a standard tool, I think we should link to the install sub-heading on the README. If I was coming into this blind, I would find the mention of this tool confusing.

**Review by jasonBirchall — CHANGES_REQUESTED** [review]
Excellent. I recommend just one teeny tiny change and then this is good to go.

File: source/documentation/deploying-an-app/github-actions-continuous-deployment.html.md.erb (hunk 1 above)
Comment (digitalronin) [reply source/documentation/deploying-an-app/github-actions-continuous-deployment.html.md.erb:311]: There is a link to [this page](https://user-guide.cloud-platform.service.justice.gov.uk/documentation/getting-started/cloud-platform-cli.html#the-cloud-platform-cli) just above the code snippets.

Screenshot 2021-01-06 at 13 15 44

Do you think that's not enough? I don't think I can make the text **inside** the code snippet into a link, unfortunately.

Comment (jasonBirchall) [reply source/documentation/deploying-an-app/github-actions-continuous-deployment.html.md.erb:311]: Okay, that should be fine.

**Review by jasonBirchall — APPROVED** [review]

**Pull request merged**